ev3dev/core.py

```diff
@@ -1814,6 +1810,46 @@ def is_pressed(self):
     def is_released(self):
         return not self.is_pressed
 
+    def _wait(self, wait_for_press, timeout_ms):
+        tic = time.time()
+
+        if self._poll is None:
+            self._value0 = self._attribute_file_open('value0')
+            self._poll = select.poll()
+            self._poll.register(self._value0, select.POLLPRI)
+
+        while True:
+            self._poll.poll(timeout_ms)
+
+            if wait_for_press:
```

Comment:
Can we reduce this to a single Boolean expression?

Reply:
Meaning `if (wait_for_press and self.is_pressed) or (not wait_for_press and self.is_released)`?  I could go either way but I think the current layout is a tiny bit easier to read.

Reply:
`wait_for_press ^ self.is_released` or `self.is_pressed == wait_for_press`? I think either should work, feel free to switch inversions for readability if that's useful.

Reply:
ah I follow you now, fixed.